tonic/transforms.py

```diff
@@ -189,7 +189,7 @@ class DropEventByArea:
     specified ratio of the sensor size.
 
     Args:
-        sensor_size (Tuple): size of the sensor that was used [W,H,P]
+        sensor_size (Optional[Tuple[int, int, int]]): size of the sensor that was used [W,H,P]. Defaults to None.
```

Comment:
Here we need to explain what setting it to None means, that it'll automatically calculate the sensor size from the maximum event locations in the data and that that is not a guarantee that it is the true sensor size